### ISSUE TITLE
make success/failure colors configurable

### ISSUE THREAD

**Issue by dchelimsky** [rspec/rspec-core#685]
Red/green colorblind users would benefit from the ability to configure success/failure colors.

**Comment by hlascelles** [comment]
It appears the answer is still no.

**Comment by myronmarston** [comment]
Things that don't have explicit CLI options can still be configured from `.rspec`; you just have to `--require` a ruby file from there, and then configure it in that file.  So, for example, if you wanted to globally change the colors on your machine, you could create a file like `~/configure_rspec_colors.rb`:

``` ruby
RSpec.configure do |c|
  # Each of these can be one of the ANSI color code integers or one
  # of [:black, :white, :red, :green, :yellow, :blue, :magenta, :cyan].
  # The values I've assigned here are the defaults so you'll want to change them.

  # Color config is available in RSpec 2.13+. Since this file will
  # always be loaded by RSpec and you may have projects on earlier
  # versions, it's a good idea to only do this on versions that support it.
  if c.respond_to?(:default_color=)
    c.default_color = :white
    c.detail_color  = :cyan
    c.failure_color = :red
    c.fixed_color   = :blue
    c.pending_color = :yellow
    c.success_color = :green
  end
end
```

And then add `--require "~/configure_rspec_colors.rb"` to `~/.rspec`.  Any time you run rspec on your machine, you'll get the custom colors.
